app/assets/javascripts/admin/order_cycles/controllers/create.js.coffee

```diff
@@ -12,7 +12,13 @@ angular.module('admin.orderCycles')
     $scope.StatusMessage = StatusMessage
 
     $scope.$watch 'order_cycle_form.$dirty', (newValue) ->
-      StatusMessage.display 'notice', 'You have unsaved changes' if newValue
```

Comment:
It looks like there are multiple concerns addressed in this commit:

- Extracting some existing strings to translations
- Displaying OC validation messages
- Requiring the OC name field

Multiple concerns in a single commit make reviewing the code harder (not too much in this case, but larger commits can be hard to untangle) and when diagnosing issues with `git bisect`, they make it harder to trace the cause of a problem. On a different project I ended up spending many days trying to track down an issue that was introduced by an epic refactor performed in one commit.

Have you had much experience using `git rebase -i`? Would you be up for splitting this commit out into those three?